src/include/verify.h

```diff
@@ -0,0 +1,27 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WT_VERIFY_INFO -- A structure to hold all the information related to a verify operation.
+ */
+struct __wt_verify_info {
+    WT_SESSION_IMPL *session;
+
+/* AUTOMATIC FLAG VALUE GENERATION START 12 */
+#define WT_VRFY_DISK_CONTINUE_ON_FAILURE 0x1000u
+#define WT_VRFY_EMPTY_PAGE_OK 0x2000u
```

Comment:
Given that these are defined in the header, could callers pass them directly to wt_vrfy_dsk_image? That is replace the last two parameters with a flags?
I also think they should share a common `WT_VRFY_DISK_` prefix, i.e: `WT_VRFY_EMPTY_PAGE_OK` should become `WT_VRFY_DISK_EMPTY_PAGE_OK`